rules/library.bzl

```diff
@@ -97,7 +97,7 @@ extend_modulemap = rule(
     doc = "Extends a modulemap with a Swift submodule",
 )
 
-def _write_modulemap(name, umbrella_header = None, module_name = None, framework = False):
+def _write_modulemap(name, umbrella_header = None, public_headers = [], private_headers = [], module_name = None, framework = False):
```

Comment:
@dierksen FYI added these back in for use of custom module maps that need to know the public/private header

Reply:
Why did this PR end up having to change the module maps? Overall I think this should probably be removed - it would just set the `modulemap` attribute of `apple_framework` better

Reply:
Agh I see what happened here - sorry for the breakage
https://github.com/bazel-ios/rules_ios/commit/0f024b4bea57bbfa6c13e5c2c53c07a1320358ff#diff-23068963449a2bf4d162d813a4dda51236b9d044d1442e8a4bb4ad7680473478L811

TBH I wonder how this got through rules_ios CI - it's something we can follow up with